eclair-core/src/test/scala/fr/acinq/eclair/io/OpenChannelInterceptorSpec.scala

```diff
@@ -47,6 +47,8 @@ class OpenChannelInterceptorSpec extends ScalaTestWithActorTestKit(ConfigFactory
   val defaultParams: DefaultParams = DefaultParams(100 sat, 100000 msat, 100 msat, CltvExpiryDelta(288), 10)
   val openChannel: OpenChannel = createOpenChannelMessage()
   val remoteAddress: NodeAddress = IPAddress(InetAddress.getLoopbackAddress, 19735)
+  val acceptStaticRemoteKeyChannelsTag = "accept static_remote_key channels"
+  val defaultFeatures = initFeatures().add(AnchorOutputsZeroFeeHtlcTx, Optional)
```

Comment:
This `initFeatures` that actually calls to the Bolt 11 spec is quite hacky and unclear, why don't you simply define a `Features` object here with the default features we expect? I think it should be cleaned up from this file, this was probably a mistake.

Reply:
done in https://github.com/ACINQ/eclair/pull/2881/commits/fccdf956aa837b698ec45071fa3a1698d3544396